src/schema/v2/artist/helpers.ts

```diff
@@ -97,6 +97,11 @@ export const ARTIST_INSIGHT_MAPPING: Record<
     getEntities: (artist) => artist.critically_acclaimed && [],
     getLabel: () => "Critically Acclaimed",
   },
+  RECENT_CAREER_EVENT: {
+    getDescription: (artist) => artist.recent_show && getRecentShow(artist),
+    getEntities: (artist) => artist.recent_show && [],
```

Comment:
is it intentional to return `null` or `[]`?

Reply:
yes